### PR TITLE
Fix: scheduler env variables

### PR DESCRIPTION
**Pull request by Meldiron**
## What does this PR do?

Fix missing env variables on functions executed over scheduler.

## Test Plan

- [x] Manual QA

## Related PRs and Issues

x

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -117,6 +117,7 @@ RUN mkdir -p /storage/uploads && \
 # Executables
 RUN chmod +x /usr/local/bin/doctor && \
     chmod +x /usr/local/bin/patch-create-missing-schedules && \
```
Comment (christyjacob4) [inline Dockerfile:119]: can you now delete the old patch file ?

Comment (Meldiron) [reply Dockerfile:119]: Just bin file, or the while patch task?
We might want to keep it for historical reasons. Allows copy&pasting when the similar patch is needed. Feels more safer than relaying on Migration files

File: src/Appwrite/Platform/Tasks/PatchDeleteScheduleUpdatedAtAttribute.php [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            }
+            $projects = $dbForConsole->find('projects', $projectsQueries);
+
+            if (count($projects) === 0) {
+                break;
+            }
+
+            foreach ($projects as $project) {
+                Console::log("Checking Project " . $project->getAttribute('name') . " (" . $project->getId() . ")");
+                $dbForProject = $getProjectDB($project);
```
Comment (christyjacob4) [inline src/Appwrite/Platform/Tasks/PatchDeleteScheduleUpdatedAtAttribute.php:53]: shouldn't this be reclaimed back in the pool ?

File: app/controllers/api/functions.php [hunk 3]
```diff
@@ -786,22 +772,6 @@
...
-
-        $active = !empty($function->getAttribute('schedule'));
-
-        if ($active) {
-            $schedule->setAttribute('resourceUpdatedAt', datetime::now());
-        }
-
-        $schedule->setAttribute('active', $active);
-
-        Authorization::skip(fn () => $dbForConsole->updateDocument('schedules', $schedule->getId(), $schedule));
```
Comment (christyjacob4) [inline app/controllers/api/functions.php:803]: why did we remove this ? How is the `active` attribute set when there is a new deployment ?

File: src/Appwrite/Platform/Tasks/PatchDeleteScheduleUpdatedAtAttribute.php (hunk 2 above)
Comment (Meldiron) [reply src/Appwrite/Platform/Tasks/PatchDeleteScheduleUpdatedAtAttribute.php:55]: Added reclaim logic.

File: Dockerfile (hunk 1 above)
Comment (christyjacob4) [reply Dockerfile:119]: hmm, okay you can keep the actual file but remove the binary

File: app/controllers/api/functions.php (hunk 3 above)
Comment (christyjacob4) [reply app/controllers/api/functions.php:803]: Okay so is the builds worker updating the schedule ?

Comment (Meldiron) [reply app/controllers/api/functions.php:803]: Correct: https://github.com/appwrite/appwrite/blob/567cb01b9940948575abb08a1654b92cc61fe6d9/app/workers/builds.php#L214

File: Dockerfile (hunk 1 above)
Comment (Meldiron) [reply Dockerfile]: Removed bin file

File: app/controllers/api/functions.php (hunk 3 above)
Comment (christyjacob4) [reply app/controllers/api/functions.php:803]: okay cool 👍

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**